doc/sphinx/src/models.rst

```diff
@@ -59,51 +73,206 @@ Davis EOS use the material's isentrope. In ths way, the reference curve
+| :math:`P(\rho, e)`        | Pressure             |                          |
++---------------------------+----------------------+                          |
+| :math:`S(\rho, e)`        | Specific Entropy     |                          |
++---------------------------+----------------------+--------------------------+
+| :math:`e(\rho, T)`        | Specific Internal    | Density, Temperature     |
+|                           | Energy               |                          |
++---------------------------+----------------------+                          |
+| :math:`P(\rho, T)`        | Pressure             |                          |
++---------------------------+----------------------+                          |
+| :math:`S(\rho, T)`        | Specific Entropy     |                          |
++---------------------------+----------------------+--------------------------+
+| :math:`\rho(P, T)` [#PT]_ | Density              | Pressure, [#PT]_         |
++---------------------------+----------------------+ Temperature [#PT]_       |
+| :math:`e(P, T)` [#PT]_    | Specific Internal    |                          |
+|                           | Energy               |                          |
++---------------------------+----------------------+--------------------------+
+| :math:`C_V(\rho, T)`      | Constant Volume      | Density, Temperature     |
++---------------------------+ Specific Heat        +--------------------------+
+| :math:`C_V(\rho, e)`      | Capacity             | Density, Specific        |
+|                           |                      | Internal Energy          |
++---------------------------+----------------------+--------------------------+
+| :math:`B_S(\rho, T)`      | Isentropic Bulk      | Density, Temperature     |
++---------------------------+ Modulus              +--------------------------+
+| :math:`B_S(\rho, e)`      |                      | Density, Specific        |
+|                           |                      | Internal Energy          |
++---------------------------+----------------------+--------------------------+
+| :math:`\Gamma(\rho, T)`   | Gruneisen Parameter  | Density, Temperature     |
++---------------------------+                      +--------------------------+
+| :math:`\Gamma(\rho, e)`   |                      | Density, Specific        |
+|                           |                      | Internal Energy          |
++---------------------------+----------------------+--------------------------+
+
+.. [#PT]
+    Note: Using pressure and temperature as independent variables is fraught
+    since both pressure and energy are often multi-valued in density for many
+    EOS due to the presence of phase changes (especially tabular EOS). For EOS
+    in ``singularity-eos`` where there is not an analytic inversion to
+    pressure-temperature space, a root-find is typically used that uses the
+    density at standard temperature and pressure (STP) as an initial guess. Thus
```

Comment:
I don't know if this is true. I think we let the user specify an initial guess don't we?

Reply:
Here is my quick survey of the `DensityEnergyFromPressureTemperature()` function in our EOS. It looks like JWL is the only analytic that allows for a user-supplied guess in the root find while all others behave as I describe or have an analytic inversion.

- Ideal gas: analytic inversion
- Gruneisen: analytic inversion for energy. Density is analytically inverted in expansion but uses a root find in compression with an ambient density guess
- JWL: Root find with user-supplied guess through the function argument if it's large enough (does not use lambda)
- Davis Reactants: Root find with ambient density guess (no user input)
- Davis Products: Root find with ambient density guess (no user input)
- Vinet: Not implemented error
- EOSPAC: Whatever EOSPAC does under the hood
- Spiner: Can utilize `lambda` for a user-supplied guess, but otherwise ignores the density function argument

I'll note that spiner can take user-supplied guesses in the lambda parameter but I'll keep the language about analytic EOS since I think it broadly applies.

Reply:
I'll create an issue to possibly unify this behavior in the base class and provide custom implementations where appropriate.

Reply:
Ah never mind, we already have an issue. I'll rename it
https://github.com/lanl/singularity-eos/issues/199